src/main/features/claimant-response/routes/check-and-send.ts

```diff
@@ -26,11 +26,13 @@ function getPaymentIntention (draft: DraftClaimantResponse, claim: Claim): Payme
     return undefined
   }
 
-  if (draft.acceptPaymentMethod && draft.acceptPaymentMethod.accept &&
-    draft.acceptPaymentMethod.accept.option === YesNoOption.YES ||
-    (draft.settleAdmitted && draft.settleAdmitted.admitted.option === YesNoOption.YES)) {
+  if ((draft.settleAdmitted && draft.settleAdmitted.admitted.option === YesNoOption.YES)
+    && (draft.acceptPaymentMethod &&
+      draft.acceptPaymentMethod.accept.option === YesNoOption.YES)) {
     return response.paymentIntention
-  } else if (claim.response.defendant.type === PartyType.INDIVIDUAL.value) {
+  } else if ((draft.settleAdmitted && draft.settleAdmitted.admitted.option === YesNoOption.YES) &&
+    (draft.acceptPaymentMethod && draft.acceptPaymentMethod.accept.option === YesNoOption.NO) &&
+    claim.response.defendant.type === PartyType.INDIVIDUAL.value) {
     return draft.courtDetermination.courtDecision
   } else {
```

Comment:
truth table here is like:
```
admitted | acceptMethod | individual | result
    0    |       0      |      0     | alternate
    0    |       0      |      1     | alternate
    0    |       1      |      0     | alternate
    0    |       1      |      1     | alternate
    1    |       0      |      0     | alternate
    1    |       0      |      1     | court
    1    |       1      |      0     | response
    1    |       1      |      1     | response
```
So this could be along the lines of:
```
IF settleAdmitted is NO
    THEN RETURN alternate payment plan
ELSE IF acceptMethod is YES
    THEN RETURN response payment plan
ELSE IF defendant is an INDIVIDUAL
    THEN RETURN court payment plan
ELSE
    RETURN alternate payment plan
```
which I hope would be more readable

Reply:
Discussed offline - agreed with @singh-surjeet to leave as is.